src/main/src/main/java/org/geoserver/ows/QuickTemplate.java

```diff
@@ -0,0 +1,47 @@
+/* (c) 2017 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.ows;
+
+import java.util.Map;
+
+/**
+ * Simpleton but fast template engine, replaces variables following the "${varName}" syntax into a
+ * string. No escaping, no extras, but avoids building lots of strings to do its work and the
+ * overhead of template instantiation of a true template engine.
+ *
+ * @author Andrea Aime - GeoSolutions
```

Comment:
@lmoran if this is a new file, please check the author tag. Was this copied from elsewhere?

Reply:
Yes, copied and, later, adapted.